### PR TITLE
Update DTA config to not require json file and switch to IMJS_ prefix

### PR DESCRIPTION
**Pull request by calebmshafer**
Switch the DTA configuration variables to all be `IMJS_` prefixed instead of `SVT_`. The `IMJS_` prefix has special meaning in our fork of react-scripts (`@bentley/react-scripts`) and only environment variables with that prefix , or the `REACT_APP_` prefix, are loaded into the frontend.

With the above change the configuration.json file can now be removed and the env vars can be consistently loaded on both the backend and frontend side without a request.

Added a couple other config vars to the main DTA config to consolidate all places `process.env` was used.

Note: You can now build only the backend (`npm run build:backend`) or just run "npm run compile" in order to be able to start the app. No need to ever run the full build unless that's your intention.

### CODE REVIEW

File: test-apps/display-test-app/src/frontend/DisplayTestApp.ts
Comment (calebmshafer) [inline test-apps/display-test-app/src/frontend/DisplayTestApp.ts:21]: @markschlosseratbentley any idea on the why?

File: test-apps/display-test-app/src/backend/Backend.ts
Comment (calebmshafer) [inline test-apps/display-test-app/src/backend/Backend.ts:56]: @markschlosseratbentley @pmconne I didn't dig too deep but which is this used for? Do we still need it?

Comment (markschlosseratbentley) [reply test-apps/display-test-app/src/backend/Backend.ts:56]: This is the Documents path on an iPad or iPhone for the specific app running.  This is the writeable directory dta will have access to.  @khanaffan added this variable, I believe... maybe he has some suggestions.

Comment (markschlosseratbentley) [reply test-apps/display-test-app/src/backend/Backend.ts:56]: FYI, in order for display-test-app to save external saved views, this is needed.  I am not sure if we will need some other way to get the Documents path than an environment variable due to the changes in this PR.

File: test-apps/display-test-app/src/frontend/DisplayTestApp.ts
Comment (markschlosseratbentley) [reply test-apps/display-test-app/src/frontend/DisplayTestApp.ts:21]: @khanaffan Is this code path needed still?

**Comment by calebmshafer** [comment]
@Mergifyio update

**Review by pmconne — APPROVED** [review]
Works on Windows. We'll need to test mobile.

File: test-apps/display-test-app/src/backend/Backend.ts
Comment (calebmshafer) [reply test-apps/display-test-app/src/backend/Backend.ts:56]: The variable should still work with this PR just won't be accessible on the frontend.

Why do you need this one to save external saved views?

Comment (pmconne) [reply test-apps/display-test-app/src/backend/Backend.ts:56]: On iOS, we need a directory to which we have permission to write.

Comment (calebmshafer) [reply test-apps/display-test-app/src/backend/Backend.ts:56]: Oh okay, thought it was to save the views to a service not just locally on the disk. Sounds good.

Do you want me to switch it to `IMJS_DOCS` or leave it as is? It'll technically work either way.

Comment (pmconne) [reply test-apps/display-test-app/src/backend/Backend.ts:56]: Safest to leave it I think. We only need it on the backend anyway.

Comment (calebmshafer) [reply test-apps/display-test-app/src/backend/Backend.ts:56]: Sounds good. Resolving...

Comment (pmconne) [reply test-apps/display-test-app/src/backend/Backend.ts:56]: Can you revert your incredulous "Why???" and "What is DOCS?" comments?

Comment (calebmshafer) [reply test-apps/display-test-app/src/backend/Backend.ts]: Updated to a more useful comment for the DOCS and removed the Why

File: test-apps/display-test-app/src/frontend/DisplayTestApp.ts
Comment (pmconne) [reply test-apps/display-test-app/src/frontend/DisplayTestApp.ts:20]: @markschlosseratbentley did you receive an answer from @khanaffan?

Comment (calebmshafer) [reply test-apps/display-test-app/src/frontend/DisplayTestApp.ts:20]: I shouldn't have broken anything here was moreso wondering if it was used. If no objections, I'll leave it as is so we can merge this PR.

Comment (pmconne) [reply test-apps/display-test-app/src/frontend/DisplayTestApp.ts:20]: Somebody needs to test it on iOS.

Comment (calebmshafer) [reply test-apps/display-test-app/src/frontend/DisplayTestApp.ts:20]: Are there any instructions, I don't see any in the readme? I'm on my Mac today and have an iPad I could use...

Comment (pmconne) [reply test-apps/display-test-app/src/frontend/DisplayTestApp.ts:20]: I always have to ask @khanaffan or @markschlosseratbentley for instructions for building and running on iOS. Better if display team tests anyway (but readily available instructions e.g. in the README would be great).

Comment (calebmshafer) [reply test-apps/display-test-app/src/frontend/DisplayTestApp.ts:20]: Figured it out, testing now. Just need to run `build:ios` and then open `test-apps/display-test-app/ios/imodeljs-test-app/imodeljs-test-app.xcodeproj/project.pbxproj` in Xcode and run it.

Comment (markschlosseratbentley) [reply test-apps/display-test-app/src/frontend/DisplayTestApp.ts:20]: **@pmconne: "@markschlosseratbentley did you receive an answer from @khanaffan?"**
No. @khanaffan Is this code path still needed?

Comment (markschlosseratbentley) [reply test-apps/display-test-app/src/frontend/DisplayTestApp.ts:20]: @calebmshafer When I try to build this for iOS (`npm run build:ios`) , I get the following error. Did you successfully build this branch?
```
ERROR in /Users/markschlosser/imodeljs/common/temp/node_modules/.pnpm/node-fetch@2.6.4/node_modules/node-fetch/lib/index.mjs 1157:15-18
Can't import the named export 'URL' from non EcmaScript module (only default export is available)
 @ /Users/markschlosser/imodeljs/common/temp/node_modules/.pnpm/@azure+core-http@2.2.0/node_modules/@azure/core-http/dist-esm/src/nodeFetchHttpClient.js
 @ /Users/markschlosser/imodeljs/common/temp/node_modules/.pnpm/@azure+core-http@2.2.0/node_modules/@azure/core-http/dist-esm/src/defaultHttpClient.js
 @ /Users/markschlosser/imodeljs/common/temp/node_modules/.pnpm/@azure+core-http@2.2.0/node_modules/@azure/core-http/dist-esm/src/coreHttp.js
 @ /Users/markschlosser/imodeljs/common/temp/node_modules/.pnpm/@azure+storage-blob@12.8.0/node_modules/@azure/storage-blob/dist-esm/storage-blob/src/index.js
 @ /Users/markschlosser/imodeljs/core/backend/lib/CloudStorageBackend.js
 @ /Users/markschlosser/imodeljs/core/backend/lib/imodeljs-backend.js
 @ ./lib/backend/Backend.js
 @ ./lib/backend/MobileMain.js
npm ERR! code ELIFECYCLE

```

Comment (calebmshafer) [reply test-apps/display-test-app/src/frontend/DisplayTestApp.ts:20]: Hmm I did not hit that error. Let me check again.

Comment (calebmshafer) [reply test-apps/display-test-app/src/frontend/DisplayTestApp.ts:20]: I just did a full clean & rebuild and didn't hit any build errors.

Comment (markschlosseratbentley) [reply test-apps/display-test-app/src/frontend/DisplayTestApp.ts:20]: Thanks Caleb... I'll try to do a full clean.

Comment (markschlosseratbentley) [reply test-apps/display-test-app/src/frontend/DisplayTestApp.ts:20]: Ok.  I did a reclone and it builds now.  However, I am running into trouble running the app (opening an iModel).  Did you successfully open an iModel, @calebmshafer ?
Once it asks me to login to my Bentley ID, I get:

```
libc++abi: terminating with uncaught exception of type NSException
dyld4 config: DYLD_LIBRARY_PATH=/usr/lib/system/introspection DYLD_INSERT_LIBRARIES=/Developer/usr/lib/libBacktraceRecording.dylib:/Developer/usr/lib/libMainThreadChecker.dylib:/Developer/Library/PrivateFrameworks/DTDDISupport.framework/libViewDebuggerSupport.dylib
*** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'Completion handler passed to -[imodeljs_test_app.ViewController webView:runJavaScriptAlertPanelWithMessage:initiatedByFrame:completionHandler:] was not called'
terminating with uncaught exception of type NSException
```

Comment (markschlosseratbentley) [reply test-apps/display-test-app/src/frontend/DisplayTestApp.ts:20]: FYI, I get the same error trying to authenticate this app using a build based on master.

Comment (calebmshafer) [reply test-apps/display-test-app/src/frontend/DisplayTestApp.ts:20]: Thanks @markschlosseratbentley. Mind if we merge this then and we can circle back to fix mobile DTA once #2152 has been merged? We should be able to get that merged tomorrow fairly early in the day.

Comment (calebmshafer) [reply test-apps/display-test-app/src/frontend/DisplayTestApp.ts:20]: @pmconne going to merge this if no objections.

Comment (markschlosseratbentley) [reply test-apps/display-test-app/src/frontend/DisplayTestApp.ts:20]: @calebmshafer Sounds fine. We will have to figure out what is going on, on both master and here, though.

**Comment by pmconne** [comment]
@Mergifyio update

**Pull request merged**
